app/src/main/java/Fragments/ProfileFragment.java

```diff
@@ -68,46 +106,66 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater menuInflater) {
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
+
         getActivity().getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_PAN);
 
         view = inflater.inflate(R.layout.fragment_profile, container, false);
 
         name_et = (TextView) view.findViewById(R.id.profile_name);
         email_et = (TextView) view.findViewById(R.id.profile_email);
         phoneNumber_et = (TextView) view.findViewById(R.id.profile_phone_number);
+        ZIP_et = (TextView) view.findViewById(R.id.profile_zip);
```

Comment:
Maybe make this lowercase to remaint consistant with naming. Else, everything looks good.

Reply:
Thanks a lot, i'll do so while coding next sprint